modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -385,6 +385,16 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         /** Returns true if there are no layers in the network. */
         CV_WRAP bool empty() const;
 
+        /** @brief Dump net to String
+         *  @returns String with structure, hyperparameters, backend, target and fusion
+         *  To see correct backend, target and fusion run after forward().
+         */
+        CV_WRAP String dumpNet();
```

Comment:
Method `dump` is a `cv::Net`'s method. So `cv::Net::dumpNet`. I think that we can name them just `dump()` and `dumpToFile`.

Reply:
done